### PR TITLE
feat(components/google-cloud): Add NAB scoring to tfp_anomaly_detection

### PR DESCRIPTION
**Pull request by g-luo**
**Description of your changes:**
- Fills in missing values from regularizing series in anomaly detection component
- Modifies visualization to account for placeholders from missing values
- Adds scoring scripts for Numenta Anomaly Benchmark (NAB)

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: components/google-cloud/google_cloud_pipeline_components/experimental/tensorflow_probability/anomaly_detection/tfp_anomaly_detection.ipynb [hunk 1]
```diff
@@ -68,8 +84,6 @@
         "\n",
         "### Dataset\n",
         "\n",
-        "{TODO: Replace synthetic data with benchmark}\n",
-        "\n",
         "This demo uses the [Numenta Anomaly Benchmark](https://github.com/numenta/NAB), a popular benchmark of time series data with labelled anomalies. More specifically, our demo uses [nyc_taxi.csv](https://github.com/numenta/NAB/blob/d2854d17a3feb9e143b1e9a715c5af67da2c1888/data/realKnownCause/nyc_taxi.csv) which reports the number of NYC taxi passengers over time.\n",
```
Comment (SinaChavoshi) [inline components/google-cloud/google_cloud_pipeline_components/experimental/tensorflow_probability/anomaly_detection/tfp_anomaly_detection.ipynb:87]: nit: s/labelled/labeled

**Review by SinaChavoshi — APPROVED** [review]
Nice work!
/lgtm

File: components/google-cloud/google_cloud_pipeline_components/experimental/tensorflow_probability/anomaly_detection/tfp_anomaly_detection.ipynb (hunk 1 above)
Comment (g-luo) [reply components/google-cloud/google_cloud_pipeline_components/experimental/tensorflow_probability/anomaly_detection/tfp_anomaly_detection.ipynb]: Done.

**Review by SinaChavoshi — APPROVED** [review]
Thank you! 
/lgtm

**Pull request merged**